src/main/java/com/dapanda/member/controller/MemberController.java

```diff
@@ -38,4 +38,15 @@ public CommonResponse<FindDataResponse> getSellingData(
 
 		return CommonResponse.success(memberService.findSellingData(userDetails.getId()));
 	}
+
+	@PostMapping("/members/profile-image")
+	public CommonResponse<Void> getProfileImage(
```

Comment:
```suggestion
	public CommonResponse<Void> updateProfileImage(
```
메서드의 기능을  생각했을 때 `updateProfileImage`가 더 맞는 것 같습니다!

Reply:
반영해서 다시 올렸습니다!